packages/dashboard/src/redux/sagas/trade.js

```diff
@@ -1,8 +1,15 @@
 import { put, takeLatest, select } from 'redux-saga/effects'
-import { getHistory, getWatchlist } from '../../utils/requests'
+import { getHistory, getWatchlist, getSecurity } from '../../utils/requests'
 import { createResponse } from '../../utils/helpers'
 import { tradeActions } from '../actions'
-import { GET_HISTORY, GET_WATCHLIST } from '../constants'
+import {
+  GET_HISTORY,
+  GET_WATCHLIST,
+  SELECT_SECURITY,
+  GET_SECURITY_HISTORY,
+} from '../constants'
+
+const getTokens = state => state.auth.user.tokens
```

Comment:
could use this on all requests, so we dont have to pass in tokens from where we dispatch the action
refactor in separate pr